src/ng/q.js

```diff
@@ -381,7 +381,11 @@ function qFactory(nextTick, exceptionHandler, errorOnUnhandledRejections) {
       if (!toCheck.pur) {
         toCheck.pur = true;
         var errorMessage = 'Possibly unhandled rejection: ' + toDebugString(toCheck.value);
-        exceptionHandler(errorMessage);
+        if (toCheck.value instanceof Error) {
+          exceptionHandler(toCheck.value, errorMessage);
```

Comment:
Hmm, OK. I mostly wondered because this:
<img width="768" alt="screen shot 2016-12-21 at 15 48 43" src="https://cloud.githubusercontent.com/assets/1758366/21393506/0a9b266c-c795-11e6-9df7-da2b8399f970.png">
looks a little weird.